BeeKit/Managers/HealthStoreManager.swift

```diff
@@ -6,22 +6,38 @@
 //  Copyright © 2017 APB. All rights reserved.
 //
 
+import CoreData
 import Foundation
 import HealthKit
 import OSLog
 
-public class HealthStoreManager :NSObject {
+public class HealthStoreManager {
+    /// The number of days to update when we are informed of a change. We are only called when the device is unlocked, so we must look
+    /// at the previous day in case data was added after the last time the device was locked. There may also be other integrations which report
+    /// data with some lag, so we look a bit further back for safety
+    /// This does mean users who have very little buffer, and are not regularly unlocking their phone, may erroneously derail. There is nothing we
+    /// can do about this.
+    static let daysToUpdateOnChangeNotification = 7
```

Comment:
Previously there was a reminder about this in the form of a local notification. It was removed with https://github.com/beeminder/BeeSwift/pull/251 .